### PR TITLE
Add input support for CIELAB (#3)

### PR DESCRIPTION
**Pull request by MusiKid**
This commit adds the support for parsing CIELab in the CLI.

### CODE REVIEW

File: src/cli/parser.rs [hunk 1]
```diff
@@ -300,6 +348,34 @@ fn parse_gray() {
...
+        parse_color("Lab(15,35.5,-43.4)")
+    );
+    assert_eq!(
+        Some(Color::from_lab(15.0, -35.5, -43.4, 0.4)),
+        parse_color("Lab(15,-35.5,-43.4,0.4)")
+    );
+    assert_eq!(
+        Some(Color::from_lab(15.0, 23.0, -43.0, 1.0)),
+        parse_color("Lab(        15,  23,-43   )")
+    );
```
Comment (sharkdp) [inline src/cli/parser.rs]: Can we also add a simple test for `CIELab(...)`?

**Comment by sharkdp** [comment]
Thank you very much, looks great!

**Comment by MusiKid** [comment]
I added more tests for the parser.

**Comment by sharkdp** [comment]
Awesome, thanks!

**Comment by MusiKid** [comment]
No problem!

**Pull request merged**